File: app/src/main/java/org/tvheadend/tvhclient/ui/features/navigation/CustomProfileDrawerItem.java
package org.tvheadend.tvhclient.ui.features.navigation;

import android.view.View;
import android.widget.TextView;

import com.mikepenz.materialdrawer.holder.StringHolder;
import com.mikepenz.materialdrawer.model.ProfileDrawerItem;

import org.tvheadend.tvhclient.R;

import java.util.List;

class CustomProfileDrawerItem extends ProfileDrawerItem {

    public void bindView(ViewHolder viewHolder, List payloads) {
        super.bindView(viewHolder, payloads);

        TextView name = viewHolder.itemView.findViewById(R.id.material_drawer_name);
        name.setVisibility(View.VISIBLE);
        StringHolder.applyTo(this.getName(), name);

        TextView email = viewHolder.itemView.findViewById(R.id.material_drawer_email);
        email.setVisibility(View.VISIBLE);
        StringHolder.applyTo(this.getEmail(), email);

        //call the onPostBindView method to trigger post bind view actions (like the listener to modify the item if required)
        onPostBindView(this, viewHolder.itemView);
    }
}

File: app/src/main/java/org/tvheadend/tvhclient/ui/features/navigation/CustomProfileDrawerItem.kt
package org.tvheadend.tvhclient.ui.features.navigation

import android.widget.TextView
import com.mikepenz.materialdrawer.holder.StringHolder
import com.mikepenz.materialdrawer.model.ProfileDrawerItem
import org.tvheadend.tvhclient.R
import org.tvheadend.tvhclient.ui.common.visible

internal class CustomProfileDrawerItem : ProfileDrawerItem() {

    override fun bindView(viewHolder: ProfileDrawerItem.ViewHolder, payloads: List<*>?) {
        super.bindView(viewHolder, payloads as List<*>)

        val name = viewHolder.itemView.findViewById<TextView>(R.id.material_drawer_name)
        name.visible()
        StringHolder.applyTo(this.getName(), name)

        val email = viewHolder.itemView.findViewById<TextView>(R.id.material_drawer_email)
        email.visible()
        StringHolder.applyTo(this.getEmail(), email)

        //call the onPostBindView method to trigger post bind view actions (like the listener to modify the item if required)
        onPostBindView(this, viewHolder.itemView)
    }
}
